docs/src/pages/system/properties/properties.md

```diff
@@ -1,60 +1,91 @@
-| [spacing](/system/spacing/)       | `spacing`        | `px,paddingX`      | `padding-left`, `padding-right`                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `py,paddingY`      | `padding-top`, `padding-bottom`                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [typography](/system/typography/) | `typography`     | `typography`       | `font-family`, `font-weight`, `font-size`, `line-height`, `letter-spacing`, `text-transform` | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontFamily`     | `fontFamily`       | `font-family`                                                                                | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontSize`       | `fontSize`         | `font-size`                                                                                  | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontWeight`     | `fontWeight`       | `font-weight`                                                                                | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `textAlign`      | `textAlign`        | `text-align`                                                                                 | none                                                                   |
+<p class="description">This page lists all custom system properties, how are they linked with the theme, and which CSS properties are they resulting in.</p>
+
+For example, let's the the following row:
+
+| Group                       | System style function | System key        | CSS property/properties | Theme key                                                        |
+| :-------------------------- | :-------------------- | :---------------- | :---------------------- | :--------------------------------------------------------------- |
+| [spacing](/system/spacing/) | `spacing`             | `mb,marginBottom` | `margin-bottom`         | [`spacing`](/customization/default-theme/?expand-path=$.spacing) |
+
+1. The <b>Group</b> column gives you access to the documentation page where this group of properties are described, in the example, the [spacing](/system/spacing/) page.
+2. The <b>System style function</b> is referencing the style function which generates these properties, in case you want to use it for adding this functionality to your custom components. The function can be imported from `@material-ui/system`.
```

Comment:
```suggestion
2. The <b>System style function</b> is referencing the style function which generates the properties in the other columns, in case you want to use it for adding this functionality to your custom components. The function can be imported from `@material-ui/system`.
```